gsl_programmation/services.py

```diff
@@ -35,8 +35,11 @@ def create_simulation(cls, user: Any, title: str, dotation: str):
                 arrondissement=None,
             )
 
-        enveloppe = Enveloppe.objects.get(
-            perimetre=perimetre_to_find, type=dotation, annee=date.today().year
+        enveloppe, _ = Enveloppe.objects.get_or_create(
```

Comment:
Peut-être qu'ici on pourrait garder la valeur de la variable indiquant si l'objet a été créé : 
```suggestion
        enveloppe, created = Enveloppe.objects.get_or_create(
```

Et un peu plus loin, si `created == True`, alors on lève une alerte à destination de la DGCL indiquant que les préfectures commencent les simulations sans avoir les montants.